tests/e2e/specs/in-game/one-offs/0_countering.spec.js

```diff
@@ -514,6 +514,7 @@ describe('Opponent May Counter vs Opponent Must Resolve', () => {
             const errors = [
               "Can't play while waiting for opponent to counter",
               'Can only draw in main phase',
```

Comment:
Can we remove the second error message here? It should only be the first one or the last one you added below since that's what it now says in the GameState API